ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeKeyStore.scala

```diff
@@ -21,64 +22,109 @@ import java.security.interfaces.RSAPublicKey
  */
 class FakeKeyStore(mkLogger: LoggerFactory) {
   private val logger = mkLogger(getClass)
-  val GeneratedKeyStore = "generated.keystore"
-  val DnName = "CN=localhost, OU=Unit Testing, O=Mavericks, L=Moon Base 1, ST=Cyberspace, C=CY"
-  val SignatureAlgorithmOID = AlgorithmId.sha256WithRSAEncryption_oid
+
+  val GeneratedKeyStore: String = fileInDevModeDir("generated.keystore")
+  val ExportedCert: String = fileInDevModeDir("service.crt")
+  val TrustedAlias = "playgeneratedtrusted"
```

Comment:
There's few `"playgeneratedXXX"` constants. 
These are not very relevant for play and are actually undocumented. I think we can rename them to somthing more generic. IIRC there are 4 different names: 

1. label for a CA
2. label for a general user certificate emit by the CA
3. label for a trusted object representing the CA
3. label for a trusted object representing the certificate created in 2.